frontend/src/components/jobs/result/api.jsx

```diff
@@ -183,3 +183,31 @@ export async function retryPlugin(jobId, pluginType, pluginName) {
   }
   return success;
 }
+
+export async function retryJob(jobId, pluginType, pluginName) {
+  let success = false;
+  try {
+    const response = await axios.patch(
```

Comment:
this component is a copy paste of the retryPlugin....we need a component which creates a new job with the same parameters as the one selected

Reply:
you need to leverage the same APIs leveraged for the creation of a normal job. Please check how IntelOwl implements the "Scan" button because basically that button should do the same

Reply:
working on it.